install/kubernetes/cilium/charts/hubble-ui/templates/validation.yaml

```diff
@@ -0,0 +1,3 @@
+{{- if and .Values.global.hubble.ui.enabled (not .Values.global.hubble.enabled) }}
```

Comment:
Did you mean `.Values.global.hubble.relay.enabled`?

Reply:
This is the ui chart.

Reply:
Yes and the ui chart requires hubble-relay no?

Reply:
As far as I know, the current version does not. To my knowledge, the used v0.5 still connects to each Hubble server directly.

Reply:
the ui chart in the master branch does point to hubble-relay by default, so i think it makes sense to declare that a dependency.

https://github.com/cilium/cilium/blob/7e328b17a0b39a130e33e91285055aa9db90020f/install/kubernetes/cilium/charts/hubble-ui/templates/deployment.yaml#L32

Reply:
fixed